manifests/machineconfigdaemon/clusterrole.yaml

```diff
@@ -19,6 +19,9 @@ rules:
   resourceNames: ["privileged"]
   resources: ["securitycontextconstraints"]
   verbs: ["use"]
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "list", "watch", "patch"]
```

Comment:
I'm not sure if it's safe to have the daemon be able to access all configmaps in all namespaces, especially with the ability to run a patch on them... would you consider either:
1. restricting the MCD to just have rbac to the configmap we need via roles, or
2. have the MCO create a copy in the MCO namespace and have the MCD have a role to read all configmaps in the MCO namespace?